i2cdrivers/src/main/java/org/openftc/i2cdrivers/QwiicLEDStrip.java

```diff
@@ -0,0 +1,190 @@
+package org.openftc.i2cdrivers;
+
+import android.graphics.Color;
+import android.support.annotation.ColorInt;
+
+import com.qualcomm.robotcore.hardware.I2cAddr;
+import com.qualcomm.robotcore.hardware.I2cDeviceSynchDevice;
+import com.qualcomm.robotcore.hardware.I2cDeviceSynchSimple;
+import com.qualcomm.robotcore.hardware.I2cWaitControl;
+import com.qualcomm.robotcore.hardware.configuration.annotations.DeviceProperties;
+import com.qualcomm.robotcore.hardware.configuration.annotations.I2cDeviceType;
+
+@I2cDeviceType()
+@DeviceProperties(name = "QWIIC LED Strip", description = "Sparkfun QWIIC LED Strip", xmlTag = "QWIIC_LED_STRIP")
```

Comment:
Is this actually called an LED Strip? I can only find reference to a Qwiic LED "stick" online.

Reply:
you are right.   It is the LED Stick.  I'll make that change